internal/services/managedhsm/key_vault_managed_hardware_security_module_resource_test.go

```diff
@@ -165,13 +165,14 @@ provider "azurerm" {
 %s
 
 resource "azurerm_key_vault_managed_hardware_security_module" "test" {
-  name                     = "kvHsm%d"
-  resource_group_name      = azurerm_resource_group.test.name
-  location                 = azurerm_resource_group.test.location
-  sku_name                 = "Standard_B1"
-  tenant_id                = data.azurerm_client_config.current.tenant_id
-  admin_object_ids         = [data.azurerm_client_config.current.object_id]
-  purge_protection_enabled = false
+  name                          = "kvHsm%d"
+  resource_group_name           = azurerm_resource_group.test.name
+  location                      = azurerm_resource_group.test.location
+  sku_name                      = "Standard_B1"
+  tenant_id                     = data.azurerm_client_config.current.tenant_id
+  admin_object_ids              = [data.azurerm_client_config.current.object_id]
+  purge_protection_enabled      = false
+  public_network_access_enabled = true
```

Comment:
This is the config for the basic test which shouldn't have `Optional` properties in it. Can you please find another config/test to use to validate this case?
```suggestion
```

Reply:
Sure, that's acceptable!